### ISSUE TITLE
Populate changeset field with better changeset comment

### ISSUE THREAD

**Issue by sfkeller** [maproulette/maproulette2#226]
When starting Edit (link) with iD or JOSM, the changeset comment needs to be enhanced with better URL encoding. And hashtags in changeset comments should be avoided - or at least put at end of comment.

Proposal with example "Missing Crosswalks of Southern Switzerland (Switzerland)" to enhance current changeset comment:

> &comment=%23maproulette%20Missing_Crosswalks_of_Southern_Switzerland_(Switzerland)

to

> &comment=Missing+Crosswalks+of+Southern+Switzerland+%28Switzerland%29+%23MapRoulette

**Comment by joto** [comment]
Maybe there should be an extra field in the challenge description that contains the changeset comment? It could be pre-set with what is currently in the comment, but the challenge admin can edit it.

**Comment by joto** [comment]
Just realized that this is even worse when the Maproulette challenge will be removed at some point, because then the name of the challenge isn't useful any more to find some context. The comment should really contain enough information to stand on its own.

**Comment by mvexel** [comment]
Agreed, I think an extra field in the wizard could be useful for that where challenge authors could explicitly state the changeset comment. I would like to have the comment have a uniform part too, so changeset analysis can uncover MapRoulette trends.

**Comment by mvexel** [comment]
This should be pretty easy to add, labeling `quick-fix`
